lib/db.js

```diff
@@ -199,6 +201,7 @@ function Db(databaseName, topology, options) {
   if (this.s.noListener) return;
 
   // Add listeners
+  const topology = client.topology;
```

Comment:
nit: if you pull it out into a variable here, why not up above, since you're using it elsewhere (storing it in `this.s` for instance)